src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -2705,7 +2705,7 @@ def __init__(self, config):
         super().__init__(config)
 
         self.bert = BigBirdModel(config)
-        self.sequence_summary = SequenceSummary(config)
+        self.dropout = nn.Dropout(config.hidden_dropout_prob)
```

Comment:
@vasudevgupta7 - I think we actually implemented this incorrectly in PyTorch. Do you remember where it comes from? I'm pretty sure `sequenc_summary` should not be used for `MultipleChoice`

Reply:
BigBirdForMultipleChoice was generated through cookiecutter. I didn’t update anything there.